src/add-detele.test.js

```diff
@@ -6,13 +6,22 @@ const addTask = require('./add-delete');
 const saveDataLocalStorage = require('./local-storage');
 const loadDataLocalStorage = require('./local-storage');
 const deleteTask = require('./add-delete');
+const editTask = require('./add-delete');
+const clearCompleted = require('./add-delete');
 
 describe('add and remove Task', () => {
   it('Add Task', () => {
     let array = [];
     array = loadDataLocalStorage();
-    array = addTask(array, 'test1');
-    array = addTask(array, 'test2');
+    array = addTask(1, 'test1');
+    saveDataLocalStorage(array);
+    expect(array).toHaveLength(1);
+  });
+
+  it('Update Task', () => {
```

Comment:
- [ ]  You should test your `editTask` method properly. For this test case, you are just checking the length of the list. You should check if the `editTask` method properly updates the property that you want to update.